control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -48,6 +48,10 @@ import (
 const (
 	// TopicPrefix is the Kafka Broker topic prefix - (topic name: knative-broker-<broker-namespace>-<broker-name>).
 	TopicPrefix = "knative-broker-"
+
+	// private annotation for changing the topic
+	// NOTE: this may go away in a future release
+	externalTopicAnnotation = "x-kafka.eventing.knative.dev/external.topic"
```

Comment:
one question I have, @pierDipi 
I think... we can also name this `kafka.eventing.knative.dev/external.topic` and skip the `x-`

I do like this as a proper feature, tbh

Reply:
+1